public/manifest.json

```diff
@@ -39,11 +39,11 @@
   "browser_action": {
     "default_icon": "icons/extension_icon.png",
     "default_title": "Start recording your test",
-    "default_popup": "popup.html"
+    "default_popup": "chromeDropdown.html"
```

Comment:
We need to revert this to popup.html, dropdown is not clear. `popup` is what it is called in google docs so let's go with that